src/sentry/lang/native/unreal.py

```diff
@@ -119,4 +119,8 @@ def merge_unreal_logs_event(unreal_logs, event):
     breadcrumbs = event['breadcrumbs']['values']
 
     for log in unreal_logs:
-        breadcrumbs.append(log)
+        breadcrumbs.append({
+            'timestamp': log.get('timestamp'),
```

Comment:
`timestamp` is currently still a required value for breadcrumbs, but this will be changed soon.